### PR TITLE
Sellers can add and view products

### PR DESCRIPTION
**Pull request by ethanschreur**
**What does this PR do?**
- This pull request allows sellers to add and view products

**Description of the task to be completed** 
- Fill out and submit an add products form
- View the added products in a table

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/login
- Enter the correct email and password and submit them
- View the dashboard page
- Press the products button
- Fill out the "add products" form and submit it
- View the added product in the products table

### CODE REVIEW

File: app/templates/seller/login.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h2>Seller Login</h2>
                 <br>
                 {{ form.hidden_tag() }}
                     {% for field in form if field.widget.input_type != 'hidden' %}
-                        <p>
+                        <p class="login-inputs">
```
Comment (alexkayabula) [inline app/templates/seller/login.html:23]: @ethanschreur this change should be in a different PR.

File: static/style.css [hunk 2]
```diff
@@ -34,8 +38,7 @@ html {
 	}
 }
 
-label,
-input {
+.login-inputs {
```
Comment (alexkayabula) [inline static/style.css:41]: @ethanschreur this change should be in a different PR.

**Review by alexkayabula — CHANGES_REQUESTED** [review]

**Comment by alexkayabula** [comment]
@ethanschreur resolve the merge conflicts.

**Comment by ethanschreur** [comment]
@alexkayabula Merge conflicts are now resolved

**Review by alexkayabula — APPROVED** [review]
Thank you
